src/scripts/app-controller.js

```diff
@@ -10,18 +11,27 @@ export default class AppController {
     this._PUSH_SERVER_URL = '';
     this._API_KEY = 'AIzaSyBBh4ddPa96rQQNxqiq_qQj7sq1JdsNQUQ';
 
+    this._applicationKeys = {
+      publicKey: EncryptionHelper.base64UrlToUint8Array(
+        'BDd3_hVL9fZi9Ybo2UUzA284WG5FZR30_95YeZJsiA' +
+        'pwXKpNcF1rRPF3foIiBHXRdJI2Qhumhf6_LFTeZaNndIo'),
+      privateKey: EncryptionHelper.base64UrlToUint8Array(
+        'xKZKYRNdFFn8iQIF2MH54KTfUHwH105zBdzMR7SI3xI')
+    };
+
     // This div contains the UI for CURL commands to trigger a push
     this._sendPushOptions = document.querySelector('.js-send-push-options');
     this._payloadTextField = document.querySelector('.js-payload-textfield');
     this._stateMsg = document.querySelector('.js-state-msg');
     this._payloadTextField.oninput = () => {
-      Promise.all([
+      /** Promise.all([
```

Comment:
By default I'd say delete rather than comment out, and in particular here I don't think this is code you'd want to bring back.

Apply to anywhere that you have commented out some code.